### PR TITLE
ADJUST1-435 Increase prison-api timeout time for fetching adjudications.

### PR DESCRIPTION
**Pull request by ldlharper**

### CODE REVIEW

File: server/config.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ export default {
         response: Number(get('PRISONER_SEARCH_API_TIMEOUT_RESPONSE', 10000)),
```
Comment (carloveo-moj) [inline server/config.ts:130]:
```suggestion
        response: Number(get('PRISONER_SEARCH_API_TIMEOUT_RESPONSE', 20000)),
```
do you also need to increase this one?

File: server/config.ts [hunk 2]
```diff
@@ -130,7 +130,7 @@ export default {
...
         deadline: Number(get('PRISONER_SEARCH_API_TIMEOUT_DEADLINE', 10000)),
       },
-      agent: new AgentConfig(Number(get('PRISONER_SEARCH_API_TIMEOUT_RESPONSE', 10000))),
+      agent: new AgentConfig(Number(get('PRISONER_SEARCH_API_TIMEOUT_RESPONSE', 20000))),
```
Comment (carloveo-moj) [inline server/config.ts:133]: I think this is for prisoner search API, think you need line 93 instead

Comment (ldlharper) [reply server/config.ts]: Thank you!

**Review by carloveo-moj — APPROVED** [review]
LGTM

**Pull request merged**